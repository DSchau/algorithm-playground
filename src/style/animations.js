import { keyframes } from 'emotion';

export const ScaleIn = keyframes`
  from {
    transform: scale(0);
  }
  to {
    transform: scale(1);
  }
`;
