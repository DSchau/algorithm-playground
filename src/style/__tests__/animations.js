import * as ANIMATIONS from '../animations';

test('it has expected animations', () => {
  expect(Object.keys(ANIMATIONS)).toMatchSnapshot();
});
