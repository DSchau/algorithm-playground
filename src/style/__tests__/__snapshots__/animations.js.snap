// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it has expected animations 1`] = `
Array [
  "FADE_IN",
  "SCALE_IN",
  "SCALE_OUT",
  "SLIDE_UP",
]
`;
